### ISSUE TITLE
Error when Library has no installed packages  (v5.1.0)

### ISSUE THREAD

**Issue by unborn-andy** [TotallyInformation/node-red-contrib-uibuilder#181]
Hello .. excellent work keeping this useful node updated and with new features ! 

I get an error during NR startup that seems to be related to not having any packages installed with the node's Library.

```
29 Jun 04:00:26 - [error] TypeError: Cannot convert undefined or null to object
    at Function.keys (<anonymous>)
    at UibPackages.updateInstalledPackageDetails (C:\Users\User\.node-red\node_modules\node-red-contrib-uibuilder\nodes\libs\package-mgt.js:297:22)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
```

Software       | Version
-------------- | -------
Node.JS        |   v16.14.0
npm            |   v8.8.0
Node-RED       |  v3.0.0-beta.3
uibuilder node |  v5.1.0
uibuilderFE    |  
OS             |  Windows 11
Browser        |   Firefox 102

**Comment by TotallyInformation** [comment]
Many thanks for raising Andy.

If you need a work around, from a server command line:

```
// go to the uibRoot folder
cd ~/.node-red/uibuilder
// Install the dummy `aaa` package
npm install aaa --save
// restart node-red
```

That should do it.

I'll get a fix out ASAP.

Steve spotted a quick fix. In package-mgt.js at line 297:

```
        await Object.keys(lsParsed.dependencies).forEach( async pkgName => {
            await this.updIndividualPkgDetails(pkgName, lsParsed)
        })
```

change to

```
        await Object.keys(lsParsed.dependencies || {}).forEach( async pkgName => {
            await this.updIndividualPkgDetails(pkgName, lsParsed)
        })
```

**Comment by TotallyInformation** [comment]
Fixed in v5.1.1 now published.
